.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+          node-version: [10.x, 12.x, 14.x]
+
+    steps:
+        - uses: actions/checkout@v2
+          with:
+            submodules: true
```

Comment:
In our other repos, I think this is set to `submodules: recursive`. Could you change this to match?

Reply:
yes. sure.